lib/tapioca/rbi/model.rb

```diff
@@ -186,11 +232,58 @@ def initialize(name, *names, visibility: Visibility::Public, sigs: [], loc: nil,
         @visibility = visibility
         @sigs = sigs
       end
+
+      sig { abstract.returns(T::Array[String]) }
+      def fully_qualified_names; end
+    end
+
+    class AttrAccessor < Attr
+      extend T::Sig
+
+      sig { override.returns(T::Array[String]) }
+      def fully_qualified_names
+        parent_name = parent_scope&.fully_qualified_name
+        names.map { |name| ["#{parent_name}##{name}", "#{parent_name}##{name}="] }.flatten
```

Comment:
Instead of using `map` and then `flatten`, simply use `flat_map`.

```ruby
names.flat_map { |name| ["#{parent_name}##{name}", "#{parent_name}##{name}="] }
```